go.mod

```diff
@@ -106,6 +106,8 @@ require (
 	pgregory.net/rapid v1.1.0
 )
 
+require github.com/confluentinc/ccloud-sdk-go-v2-internal/networking-dnsforwarder v0.0.2 // indirect
```

Comment:
Initial implementation based off of existing minispec, SDK and exact command format will be updated once available

Reply:
Is this available yet?

Reply:
Not yet, was planning on updating the SDK in a separate PR once it's out.

Reply:
Let's integrate it into this PR. It's bad practice to be referring to internal repositories in a public repository, by the way.

Reply:
@brianstrauch want to confirm the usual process here. Since this feature directly moves to GA, the API SDK won't be public before GA, which means CLI changes won't be released at GA. is that ok, to release the CLI changes via next CLI release following the GA?

Reply:
> is that ok, to release the CLI changes via next CLI release following the GA?

Sure!